v1/api/swagger/components/schemas/VolumeCollection.yaml

```diff
@@ -22,3 +22,12 @@ properties:
   Description:
     type: string
     description: The description of the volume collection resource
+  PoolIDs:
+    description: Pool IDs that are valid for this volume collection
```

Comment:
IDs of the storage pools that are valid for this volume collection as listed in the Storage Pools array